services/storage/src/simcore_service_storage/redis.py

```diff
@@ -20,7 +21,9 @@ async def _setup(app: web.Application):
         assert settings.STORAGE_REDIS  # nosec
         redis_settings: RedisSettings = settings.STORAGE_REDIS
         redis_locks_dsn = redis_settings.build_redis_dsn(RedisDatabase.LOCKS)
-        app[_APP_REDIS_KEY] = client = RedisClientSDK(redis_locks_dsn)
+        app[_APP_REDIS_KEY] = client = RedisClientSDK(
+            redis_locks_dsn, client_name=APP_NAME
```

Comment:
MINOR: dont you want to distinguish between replicas as well? we might want to capture that in the app-name perhaps and then have it for free everywhere

Reply:
not sure about this fact. maybe sometimes yes, and sometimes no